### ISSUE TITLE
afterPluginsInitialized does not work

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/handsontable#4196]
### Description

`afterPluginsInitialized `does not work

### Steps to reproduce

1. Check the console. You should get a log `done` after loading all plugins (like `columnSorting`).

### Demo

http://jsfiddle.net/handsoncode/r79b0Lx7/

### Your environment
* Handsontable version: 0.31.2 / 1.10.2
* Browser Name and version: Chrome 56
* Operating System: Windows 10

**Reported at** https://forum.handsontable.com/t/cant-add-callback-for-afterpluginsinitialized-hook/918

**Comment by Rex90** [comment]
I agree - this doesn't work.

**Comment by AMBudnik** [comment]
The callback still dos not work (updated 0.35.1)

**Comment by AMBudnik** [comment]
It doesn't work in v `2.0.0` or any version before. Maybe we're missing something here? If it's not a bug we need to describe it in the docs.

**Comment by talamaska** [comment]
in version 7.0.2 still afterPluginsInitialized is not fired

**Comment by talamaska** [comment]
@AMBudnik Could you please have someone look into this issues it is reported 2 years ago and the bug is still valid

**Comment by AMBudnik** [comment]
You're right it still doesn't work http://jsfiddle.net/handsoncode/b4mh3sav/ (v `7.0.2`) and it's broken for Vanilla. If it doesn't work for vanilla it won't work for the wrappers.

I see it is called in the source code. I've built a plain example and run in on a local server and the hook is called.

![image](https://user-images.githubusercontent.com/10757813/57610763-afa39b80-7571-11e9-88f8-e66aa2b15883.png)

But it never alerts me about it

```js
var hot = new Handsontable(box, {
    data: [[1,3],[5,7]],
    licenseKey: 'non-commercial-and-evaluation',
    columnSorting: true,
    afterPluginsInitialized: function(){
      alert('afterPluginsInitialized')
    }
})
```

**Comment by wojciechczerniak** [comment]
That's because this hook is run internally on `beforeInit` hook, before settings are used. 
You can add hook before Handsontable is initialized: http://jsfiddle.net/qo2weksy/2/

Might work better after #5933.

**Comment by AMBudnik** [comment]
Hey @talamaska please try @wojciechczerniak approach and let me know if it works for you

**Comment by AMBudnik** [comment]
Issue moved to [handsontable/docs #49](https://github.com/handsontable/docs/issues/49) via [**ZenHub**](https://www.zenhub.com/)
